lib/tool_shed/webapp/config.py

```diff
@@ -101,10 +101,10 @@ def _process_config(self, kwargs):
         # Galaxy flavor Docker Image
         self.user_activation_on = None
         self.registration_warning_message = kwargs.get('registration_warning_message')
-        self.blacklist_location = kwargs.get('blacklist_file')
-        self.blacklist_content = None
-        self.whitelist_location = kwargs.get('whitelist_file')
-        self.whitelist_content = None
+        self.email_domain_blocklist_location = kwargs.get('email_domain_blocklist_file')
```

Comment:
If we rename the property to match the name in the schema, then no, we don't need to fetch it again (we shouldn't, really).

I'm not sure why those 4 lines (104-107) exist at all. I assume the TS app does NOT access `galaxy/security/validate_user_input`? And if so, those files are never used in TS.

If these lines *are*  needed, we can move [this](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L491) into [this](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L317).

Separate PR maybe?

Reply:
Thanks, I've updated my suggested changes to now remove the lines defining the 2 `_location` attributes, which seems to not be used anywhere. Not sure about the `_content` attributes.

Reply:
I'm +1 on your suggested edit.
I think both `*_content` attributes are not needed: they were introduced in 3340d6c when TS was part of `lib/galaxy`. However, I think it may be easier/more fitting to make that change separately, with some testing (along with other TS config cleanup - I suspect there's more like this).

Reply:
The TS tests were failing before these changes due to the reference in `validate_user_input`. Are you sure this is unnecessary?

Reply:
Not sure at all, now that you mention this! Let's leave as is; I'll look at this now and, once this PR is merged, I'll have this cleaned up.

Reply:
@natefoo `validate_email()` in the `validate_user_input` module uses the `_content` attributes, so the `_location` attributes can be removed already.